phoenix-spark-base/phoenix5-spark/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>phoenix-spark-base</artifactId>
+        <groupId>org.apache.phoenix</groupId>
+        <version>6.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>phoenix5-spark</artifactId>
+    <name>Phoenix5 - Spark</name>
+
+    <properties>
+        <top.dir>${project.basedir}/..</top.dir>
+        <phoenix.version>${phoenix-five.version}</phoenix.version>
+        <hbase.version>${hbase-two.version}</hbase.version>
+        <hadoop.version>${hadoop-three.version}</hadoop.version>
+        <phoenix.compat.version>5</phoenix.compat.version>
+        <curator.version>4.0.0</curator.version>
+        <jdk.version>1.8</jdk.version>
+        <maven-shade-plugin.version>3.1.1</maven-shade-plugin.version>
+        <jetty.version>9.3.19.v20170502</jetty.version>
+        <fasterxml.jackson.version>2.10.0</fasterxml.jackson.version>
+        <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
+        <javax.version>3.1.0</javax.version>
+    </properties>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-testing-util</artifactId>
+            <scope>test</scope>
+            <optional>true</optional>
+        </dependency>
+
+        <!--Jackson dependency-->
```

Comment:
I've tested locally without these dependencies, and the tests run just fine.
Please re-evaluate the additional dependencies for Phoenix 5.

The same applies for the other subprojects.